Bitcoin-Design-Crits.md

```diff
@@ -0,0 +1,48 @@
+# Bitcoin Design Crits
+
+## What are Bitcoin Design Crits?
+
+Bitcoin Design Crits are opportunities for bitcoin products to get design critiques (crits) on a particular page or section in their product. Actionable suggestions for improving usability, user interface, interaction, and visual aesthetics are given by members in the Bitcoin Design Community.
+
+## How does it work?
+
+1. A bitcoin product submits a page or section they want reviewed on Bitcoin Design Crits in the #design-review in our Slack.
+
+We will only focus on a particular page or section of your product so we can give the highest amount of value. Reviewing a whole product is an extensive and involved process that is better suited to be dealt with by a dedicated Product Designer. If you need a Product Designer for your bitcoin product, reach out in the opportunities channel in our Slack.
+
+2. A product is selected for the next upcoming Bitcoin Design Crit call by members of the community.
+
+3. Product designers interested in reviewing jump into a collaborative Figma file where they crit the product page or section submitted **PRIOR** to the Bitcoin Desisgn Crit call. This will all be public and open for anyone to view in classic open-source fashion.
```

Comment:
```suggestion
3. Product designers interested in reviewing jump into a collaborative Figma file **PRIOR** to call, where they crit the product page or section submitted. This will all be public and open for anyone to view, in classic open-source fashion.
```
Edit for clarity. Was not clear whether the submission or the crit'ing should happen before the call.